packages/adapter-cloudflare/index.d.ts

```diff
@@ -30,6 +30,12 @@ export interface AdapterOptions {
 		 */
 		exclude?: string[];
 	};
+
+	/**
+	 * The fallback page to generate (e.g. 404.html, 200.html)
+	 * See https://kit.svelte.dev/docs/adapter-static#spa-mode-add-fallback-page
+	 */
+	fallback?: string;
```

Comment:
Does this need to be a string? AFAICT from the docs, it _has_ to be `404.html`, so presumably it should be a boolean instead

Reply:
It could either be 404.html to handle 404s, or 200.html to be a SPA. Although if you want a SPA app you might as well just use static, but maybe there is a usecase where you would want a SPA except for a few api endpoints or something

Reply:
> I think it would still be a good idea to keep this as a string if they do add support for the in the future.

If they add support in future, _then_ we could change the API, but until then it can only cause confusion.

> I also wonder what would happen if you try to specify index.html as a fallback

Seems like it would get confused between the fallback page and `/`?

Reply:
> > I also wonder what would happen if you try to specify index.html as a fallback
> 
> Seems like it would get confused between the fallback page and `/`?
My understanding is that the fallback page is basically just a page with ssr disabled, so if you go to / wouldn’t it csr the correct page?

I’m deploying a project to test it. Just waiting for an ongoing cloudflare issue with new deployments to be resolved (https://www.cloudflarestatus.com/incidents/cx215c2tx7yw)

Reply:
If `/` was prerendered, you'd get that prerendered content for every 404 (rather than the blank fallback page), no?

Reply:
> If `/` was prerendered, you'd get that prerendered content for every 404 (rather than the blank fallback page), no?

It appears that it overwrites index.html, so I don’t think it would. Still waiting on cloudflare to see for sure

Reply:
Yeah, setting the fallback to index.html works.

https://cf-adapter-test.pages.dev/
(This is with fallback set to index.html)